### DISCUSSION TITLE
Documentation for self-made scripts

### DISCUSSION THREAD

**Discussion by PjotrBrunain** [cc-tweaked/CC-Tweaked#1735] [Q&A]
Hey, Is there an easy way to generate documentation for self made scripts based on comments? something like LuaDoc? I've tried getting LuaDoc to work on the computers... but haven't gotten it to work.

**Comment by SquidDev** [comment]
> I've tried getting LuaDoc to work on the computers

Yeah, LuaDoc/LDoc both indirectly rely on native libraries, so won't work (without modifications) on CC itself. You can install this outside of CC (using luarocks or something) and it should work fine!

CC:T itself uses [illuaminate](https://github.com/Squiddev/illuaminate), but it's not especially well documented, so I'd not recommend unless you're happy to fiddle and work things out for yourself :p.
